tests/vmi_multus_test.go

```diff
@@ -304,7 +307,7 @@ var _ = Describe("[Serial]Multus", func() {
 				tests.WaitUntilVMIReady(vmiOne, console.LoginToAlpine)
 
 				By("Configuring static IP address to ptp interface.")
-				Expect(configInterface(vmiOne, "eth0", "10.1.1.1/24")).To(Succeed())
+				Expect(configInterface(vmiOne, "eth0", ptpSubnet)).To(Succeed())
```

Comment:
AFAIK you can't set a subnet as the IP here.

Reply:
This is what was [been used](https://github.com/kubevirt/kubevirt/blob/master/tests/vmi_multus_test.go#L307) till now, I just moved it behind a constant. 
I think it [works also](https://linuxize.com/post/linux-ip-command/#assign-ip-addresses-to-an-interface) works subnet, but I don't mind change that as well.
Do you think I should, and if so, should I change it in this commit/PR, or in another one?

Reply:
Before a CIDR was used with an address of 10.1.1.1, now you changed it to a network address (not a host address) of 10.1.1.0.

Reply:
Ohhh Dam. My bad. 
changed back to `"10.1.1.1/24"`